src/main/resources/migrations/V1.2.0__files_view.sql

```diff
@@ -0,0 +1,21 @@
+-- Add a new column to separate the extension from the name
+ALTER TABLE archives ADD COLUMN extension VARCHAR(16) NOT NULL DEFAULT '';
```

Comment:
I believe you must delete the default value after adding the new column to the existing rows.

Reply:
Thanks